### PR TITLE
Adding quickstart related code

### PR DESCRIPTION
**Pull request by rportilla-databricks**

### CODE REVIEW

**Comment by tnixon** [comment]
I'm thinking we just point users to the notebook instead of duplicating code here... thoughts?

**Review by tnixon — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -33,113 +33,109 @@ The entry point into all features for time series analysis in tempo is a TSDF ob
...
-partitionCols = partition columns used for the range stats windowing in Spark
-```
+rangeBackWindowSecs = number of seconds to look back
 
-from tempo import *
-
-base_trades = TSDF(skewTrades, ts_col = 'event_ts')
-res_stats = base_trades.withRangeStats(partitionCols=['symbol']).df
+```
+moving_avg = watch_accel_tsdf.withRangeStats("y", rangeBackWindowSecs=600).df
```
[the remark this answers is not in the document]
Comment (rportilla-databricks) [reply README.md:137]: @tnixon , we don't have an SMA function. The way to get SMA is with range stats (mean is one of the metrics).

File: README.md [hunk 2]
```diff
@@ -33,113 +33,109 @@ The entry point into all features for time series analysis in tempo is a TSDF ob
...
-Parameters: 
+```
+from pyspark.sql.functions import * 
 
-ts_col = timestamp column on which to sort fact and source table
-partition_cols - columns to use for partitioning the TSDF into more granular time series for windowing and sorting
+phone_accel_df = spark.read.format("csv").option("header", "true").load("dbfs:/home/tempo/Phones_accelerometer").withColumn("event_ts", (col("Arrival_Time").cast("double")/1000).cast("timestamp")).withColumn("x", col("x").cast("double")).withColumn("y", col("y").cast("double")).withColumn("z", col("z").cast("double")).withColumn("event_ts_dbl", col("event_ts").cast("double"))
 
-```
+series = phone_accel_df.toPandas()
```
[the remark this answers is not in the document]
Comment (rportilla-databricks) [reply README.md:47]: The pandas data frame is used for visualization. I'll move it to right before the visualization

[the remark this answers is not in the document]
Comment (rportilla-databricks) [reply README.md]: Nevermind, this one is not needed, I removed it thanks!

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tnixon) [reply README.md:135]: Right, makes sense!

**Review by tnixon — APPROVED** [review]
Looks Good!

**Pull request merged**